assets/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,56 @@
+$base-font-face: "acumin-pro", helvetica, arial, sans-serif;
+$condensed-font-face: "acumin-pro-extra-condensed", sans-serif;
```

Comment:
Should we be using the `!default` convention for most (or all) variables in this package? These two rules basically imply a dependency on Typekit, which isn't necessarily stipulated here. Perhaps that is part of the DDK but I like the flexibility that `!default` could afford us, down the road.